neqo-transport/src/recovery.rs

```diff
@@ -605,21 +606,34 @@ pub(crate) struct LossRecovery {
 }
 
 impl LossRecovery {
-    pub fn new(stats: StatsCell) -> Self {
+    #[cfg(test)]
+    pub fn new(alg: &CongestionControlAlgorithm) -> Self {
```

Comment:
Is it so hard for the tests to provide an extra `StatsCell::default()` argument?  I'm concerned that this function and the next now need to be synchronized, all to save a tiny bit of typing (or copy-paste) in tests.

Reply:
I will remove new_with_stats.